docs/customization.md

```diff
@@ -1,8 +1,8 @@
 # How do I exclude recommendations?
 It is not uncommon for a recommendation to need to be excluded from configurations or slightly adjusted. These resources have built-in support for these situations.
-Every resource contains an 'ExcludeList' property that allows you to exclude settings based on their recommendation ID. These IDs can be found in the CIS documentation available from [CIS](src/../cis_documentation.md) or the [documentation page](https://github.com/techservicesillinois/SecOps-Powershell-CISDSC/wiki/Benchmark-documentation) of this repository. Using these PDFs you should be able to identify your setting in question and locate its ID number. Enabling exclusions will look similar to the below example.
+Every resource contains an 'ExcludeList' property that allows you to exclude settings based on their recommendation ID. These IDs can be found in the CIS documentation available from [CIS](src/../cis_documentation.md). Using these documents you should be able to identify your setting in question and locate its ID number. Enabling exclusions will look similar to the below example.
 
-```
+```powershell
```

Comment:
Actually, since this example includes both customizations and exclusions (which I would argue are a specific type of customization), why not include the sample code at the top of the article and reference it below? Otherwise it seems that the code applies only to exclusions.

Reply:
Fixed in latest push, this example is dated and duplicated from the new generated docs. I've updated it to reference those documents.